oak_abi/build.rs

```diff
@@ -15,9 +15,7 @@
 //
 
 fn main() {
-    // Exclude generation of service code, as it would require a reference to the Oak SDK to
-    // compile.
-    oak_utils::compile_protos_without_services(
+    oak_utils::compile_protos_internal(
```

Comment:
Naming-wise, it feels a bit odd that we need the `_internal_` variant to generate code in our ABI definition crate.